python/src/equistore/operations/abs.py

```diff
@@ -0,0 +1,65 @@
+"""
+Module to find the absolute values of a :py:class:`TensorMap`, returning a new
+:py:class:`TensorMap`.
+"""
+from ..block import TensorBlock
+from ..tensor import TensorMap
+from . import _dispatch
+
+
+def abs(A: TensorMap) -> TensorMap:
+    """
+    Return a new :py:class:`TensorMap` with the same metadata as A and absolute
+    values of `A`.
+
+
```

Comment:
It would be good to have the maths here - it's not completely trivial what happens to the gradients upon taking the absolute so describing (at least with using a math block) what happens in the operation would be useful.